src/js/view/controls/components/menu/menu-item.js

```diff
@@ -35,3 +35,15 @@ export class ButtonMenuItem extends MenuItem {
         this.active = false;
     }
 }
+
+export class ResetMenuItem extends MenuItem {
+    constructor(_content, _action, _template = itemButtonTemplate) {
+        super(_content, _action, _template);
+        this.statusEl = document.createElement('span');
+        this.statusEl.id = 'reset-status-message';
+        this.statusEl.style = 'font-size: 0;';
+        this.statusEl.setAttribute('role', 'status');
+        this.el.appendChild(this.statusEl);
+        this.el.setAttribute('aria-controls', 'reset-status-message');
```

Comment:
This isn't what aria-controls is for, see examples section here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls